plugins/available/basher.plugin.bash

```diff
@@ -0,0 +1,12 @@
+cite about-plugin
+about-plugin 'initializes basher, the shell package manager'
+
+# https://github.com/basherpm/basher
+
+if ! _command_exists basher && [[ -x "$HOME/.basher/bin/basher" ]]; then
```

Comment:
I think you need to do something like this:
```bash
if ! _command_exists basher ; then
	if [[ -x "$HOME/.basher/bin/basher" ]]; then
		pathmunge "$HOME/.basher/bin"
	else
		return 0
	fi
fi
```
otherwise if the `basher` command is already found, it will exit early

Reply:
ooo good catch. I totally missed that in my haste!

Reply:
Its totally okay, happens to all of us :smile: 
Thank you for the effort you are putting out here!